config/locales/en/images.yml

```diff
@@ -9,6 +9,9 @@ en:
       delete_button: Remove image
       note: "You can upload one image of following content types: %{accepted_content_types}, up to %{max_file_size} MB."
       add_new_image: Add image
+      title_placeholder: Add a descriptive title for the image
```

Comment:
This `title_placeholder` is already present at line 7 of this file, maybe the fact that [the `es` version](https://github.com/wairbut-m2c/consul/blob/master/config/locales/es/images.yml) has a different order tricked you into adding it on the same position. I would go with the same order that the `es` version has to unify both 👍